### DISCUSSION TITLE
Midjourney character reference

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#2530] [Ideas]
Please catch up. https://petapixel.com/2024/03/12/you-can-now-recreate-the-same-character-across-different-ai-images-on-midjourney/

**Comment by mashb1t** [comment]
also see https://github.com/lllyasviel/Fooocus/issues/2514
As Fooocus is mostly a hobby project for me I'm most likely not going be able to implement this soon (if there even is a paper available with the feature), so @lllyasviel maybe you have time again somewhen. Maybe you, the community, can help us here as this is a collaborative project. Thanks!

**Reply by ILikeToasters** [reply]
I have been playing around with the Midjourney feature and can't say it is any better than what can be done in Fooocus using the image prompts. But that is just my opinion so far.  The Midjourney feature is far from perfect.

**Comment by Rogala** [comment]
> Please catch up. https://petapixel.com/2024/03/12/you-can-now-recreate-the-same-character-across-different-ai-images-on-midjourney/

[This video may help, not 100% but very similar to Midjourney functionality](https://www.youtube.com/watch?v=BYL1Kf7NjYU)

**Reply by aolko** [reply]
Great, but what if i want face and body, or just body?

**Reply by Rogala** [reply]
[You can also look at this manual, a full-length reference model is already considered here](https://www.youtube.com/watch?v=weDcFFwSzIA)

AI is mostly random, it’s very difficult to get something identical, you can customize it, but no more
